src/store/modules/util/actions.ts

```diff
@@ -0,0 +1,80 @@
+import { ActionTree } from "vuex";
+import RootState from '@/store/RootState';
+import UtilState from "./UtilState";
+import * as types from "./mutation-types"
+import { hasError } from "@/utils";
+import "moment-timezone";
+import { UtilService } from "@/services/UtilService";
+
+const actions: ActionTree<UtilState, RootState> = {
+  async fetchShipmentMethods({ state, commit }) {
+    try {
+      if (Object.keys(state.shipmentMethods).length === 0) {
+        const resp = await UtilService.getShipmentMethods({
+          "inputFields":{
+            "partyId": "_NA_",
+            "roleTypeId": "CARRIER",
+            "productStoreId": this.state.user.currentEComStore.productStoreId
+          },
+          "entityName": "ProductStoreShipmentMethView",
+          "fieldList": ["shipmentMethodTypeId", "description"],
+          "noConditionFind": "Y",
+          "viewSize": 100
+        });
+
+        if(resp.status === 200 && !hasError(resp) && !(typeof resp.data === 'string' && resp.data.includes('error')) && resp.data?.docs?.length > 0) {
+          commit(types.UTIL_SHIPMENT_METHODS_UPDATED, resp.data.docs);
+        }
+      }
+    } catch(err) {
+      console.error(err);
+    }
+  },
+  async fetchCountryOptions({ state, commit }) {
+    try {
+      const resp = await UtilService.getCountryOptions({
+        "inputFields":{
+          "geoTypeId": ['COUNTRY'],
```

Comment:
```suggestion
          "geoTypeId": 'COUNTRY',
```

Reply:
Passing a single variable in the typeId when getting country list.